src/status_im/contexts/profile/contact/share/view.cljs

```diff
@@ -0,0 +1,53 @@
+(ns status-im.contexts.profile.contact.share.view
+  (:require [legacy.status-im.ui.components.list-selection :as list-selection]
+            [quo.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.qr-codes.view :as qr-codes]
+            [status-im.contexts.profile.contact.share.style :as style]
+            [status-im.contexts.profile.utils :as profile.utils]
+            [utils.address :as address]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn view
+  []
+  (let [{:keys [universal-profile-url customization-color]
+         :as   profile} (rf/sub [:contacts/current-contact])
+        abbreviated-url (rn/use-memo (fn []
+                                       (address/get-abbreviated-profile-url universal-profile-url))
+                                     [universal-profile-url])
+        profile-picture (rn/use-memo #(profile.utils/photo profile) [])
+        display-name    (rn/use-memo #(profile.utils/displayed-name profile) [])
+        on-back-press   (rn/use-callback #(rf/dispatch [:navigate-back]))
```

Comment:
do we need these functions with no dependencies wrapped in use-callback? 
cc @cammellos - I recently saw you comment on something similar

Reply:
I don't think so, this one can just be declared outside the component.

`(defn navigate-back []
    (rf/dispatch...`)

Reply:
right, yeah this is much better. We should probably update the guidelines to mention this instead 👍 
I'm seeing this in tonnes of prs now